### DISCUSSION TITLE
While installing homebrew, I used an incorrect command and now "eval" is printed every time I open the Terminal app

### DISCUSSION THREAD

**Discussion by 0rphee** [Homebrew/discussions#2042] [Getting started] [answered]
So I was installing homebrew, and while going through the final two commands, I accidentally typed `echo echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/roger/.zprofile`, note the double `echo`. Now when I open Terminal I get this, and well, I want to get rid of it...
<img width="355" alt="Screen Shot 2021-08-26 at 12 32 30 a m" src="https://user-images.githubusercontent.com/79347623/130906678-c545ff10-f25c-4acc-b679-7b71572d1042.png">
(1 eval is printed for each time I incorrectly used the `echo echo etc ...` command, for a total of 5 times (I have corroborated this in the .zsh_history file)

I think first three times I called the command was accidentally, the last two were in order to find out what caused the problem.

Keep in mind that I'm not at all an expert, I may be missing something and though I've already searched the internet for help, I haven't found anything yet. I've restarted the computer and tried to reset Terminal settings with no  results. If it is of importance, it's an M1 model. Thank you in advance.

**Comment by carlocab** [comment, marked as answer]
Edit `~/.zprofile` and remove the commands that just do `echo eval ...`. You can do this with any one of the following:
```
open -a TextEdit ~/.zprofile
nano ~/.zprofile
vim ~/.zprofile
```

**Reply by 0rphee** [reply]
Thank you so much! it worked just fine!
